executor/executorInstanceBuilder.go

```diff
@@ -0,0 +1,23 @@
+package executor
```

Comment:
Consider renaming the file to `instanceBuilder.go`. It's already in the package `executor`.

Reply:
I prefixed them all, it can be sometimes hard otherwise to seek for the file in the IDE